apps/tutorial_servo_control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,31 @@
+#include "TutorialServo.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float minPulsewidthInMs , float maxPulsewidthInMs){
+        m_servoRangeInDegrees =  servoRangeInDegrees;
+        m_minPulsewidthInMs = minPulsewidthInMs;
+	m_maxPulsewidthInMs = maxPulsewidthInMs;
```

Comment:
same here not sure if this is just github but these lines aren't aligned. If you haven't ran clang formatting it should fix it quickly.

Reply:
use the member initalizer list to initalize member variables of classes